### PR TITLE
[libalkimia]Update libalkimia to 8.1.72

### LINKED ISSUE #36527: [libalkimia] update to 8.1.72
### Library name

libalkimia

### New version number

8.1.72

### Other information that may be useful (release notes, etc...)

https://github.com/KDE/alkimia/releases/tag/v8.1.72

### PR DESCRIPTION
**Pull request by ldelabre**
Fixes #36527

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [ ] The "supports" clause reflects platforms that may be fixed by this new version.
- [ ] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by ldelabre** [comment]
@microsoft-github-policy-service agree

File: ports/libalkimia/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "libalkimia",
-  "version": "8.1.70",
-  "port-version": 2,
+  "version": "8.1.72",
+  "port-version": 3,
```
Comment (JonLiu1993) [inline ports/libalkimia/vcpkg.json:4]:
```suggestion
```

Comment (JonLiu1993) [reply ports/libalkimia/vcpkg.json:4]: When we update the port, we should remove "port-version".

File: ports/libalkimia/portfile.cmake [hunk 2]
```diff
@@ -1,12 +1,9 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO KDE/alkimia
-    REF 595186bee8409f30e5db091fffa245fc53ad92e8
-    SHA512 509082e22bc0a2ce0586e1167df14fd42ac85321315c1ee2914f60e695d1e2e8beae4fc93d16d0053edb520fc391a3dbe30777638285b295e761ad70512688ca
+    REF 8f21e4f5154f6d390609174783705ccc0559dc59
```
Comment (JonLiu1993) [inline ports/libalkimia/portfile.cmake:4]:
```suggestion
    REF "v${VERSION}"
```

**Comment by JonLiu1993** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

File: ports/libalkimia/portfile.cmake (hunk 2 above)
Comment (ldelabre) [reply ports/libalkimia/portfile.cmake:4]: I'm getting an error when I remove SHA512 :
```
CMake Error at scripts/cmake/vcpkg_from_github.cmake:12 (message):
  SHA512 must be specified if REF is specified.
```

**Comment by JonLiu1993** [comment]
Please run command "./vcpkg x-add-version libalkimia --overwrite-version" and commmit again.

**Comment by ldelabre** [comment]
Hello Jon,
My apologies for the mess with this PR - only my second on github ;)
But after some commits it seems to be ok now.
Have a nice day
Ludovic.

**Comment by JonLiu1993** [comment]
All features are tested successfully in the following triplet:
- x86-windows
- x64-windows

**Review by JonLiu1993 — APPROVED** [review]

**Pull request merged**